src/zonetree/answer.rs

```diff
@@ -40,28 +41,35 @@ pub struct Answer {
 
     /// The optional authority section to be included in the answer.
     authority: Option<AnswerAuthority>,
+
+    /// Should the answer be flagged as authoritative?
+    authoritative: bool,
 }
 
 impl Answer {
     /// Creates an "empty" answer.
     ///
-    /// The answer, authority and additinal sections will be empty.
+    /// The answer, authority and additional sections will be empty.
```

Comment:
You might also want to mention that the answer will be treated as non-authoritative.

Reply:
Done. Please check.